src/components/Table/hoc/withTableSorting/withTableSorting.tsx

```diff
@@ -40,7 +41,7 @@ export function withTableSorting<I extends TableDataItem, E extends {} = {}>(
     const displayName = `withTableSorting(${componentName})`;
 
     function defaultCompareFunction(itemA: I, itemB: I, columnId: string) {
-        if (itemA[columnId] === itemB[columnId]) {
+        if (_get(itemA, columnId) === _get(itemB, columnId)) {
```

Comment:
What was the problem here?

Reply:
for example: 
in the table we use Item with type: `{foo: {bar: string}, foo2: string, foo3: string}`
for columns Ids we use `['foo.bar', 'foo2', 'foo3']`

and right now, if we want sort table by first column, in the if we will get `undefined` from both items. Adding `_get` solve this issue

Reply:
Then it looks like new feature, not a fix

```
feat(Table): add support for sort by nested properties
```

Reply:
It seem it will be useful to add nested properties to stories and tests.